_data/cuatrimestre.yml

```
@@ -1,10 +1,10 @@
 # Primeros cuatrimestres terminan el primero de agosto
 # Segundos Cuatrimestres terminan el primero de febrero
-fin_cuatrimestre: 2021-04-01
-inicio_cuatrimestre: 2020-09-28
+fin_cuatrimestre: 2021-07-01
```

SIU dice que termina 31/07/2021
```suggestion
fin_cuatrimestre: 2021-08-01
```